pom.xml

```diff
@@ -280,6 +280,18 @@
 						<rulesDirectory>etc/jqassistant</rulesDirectory>
 						<skip>${skipArchitectureTests}</skip>
 					</configuration>
+					<dependencies>
+						<dependency>
+							<groupId>org.jqassistant.contrib.plugin</groupId>
+							<artifactId>jqassistant-dashboard-plugin</artifactId>
+							<version>1.7.0</version>
+						</dependency>
+						<dependency>
+							<groupId>de.kontext-e.jqassistant.plugin</groupId>
+							<artifactId>jqassistant.plugin.git</artifactId>
+							<version>1.6.1</version>
```

Comment:
Why not also version 1.7.0? So we could refer to the `jqassistant(.plugin).version` here and in line 287.